app/views/users/index.html.erb

```diff
@@ -12,13 +12,13 @@
 
 <ul class="users">
   <div class="search_feature">
-    <form action="" id="form1">
+    <form action="" id="form1" onsubmit="return false">
         <div>
             <input size="42" placeholder="Enter email address to search" type="text" id="search">
```

Comment:
Can they search for a name too?  This text should probably reflect that

Reply:
Good point!

Reply:
The search seems to just find a string on the page. Honestly probably a new TODO item

Reply:
For now, there is not a lot of user accounts and our clients do not seem to have a problem with it. But we can definitely create a new issue and tagged it as low priority.

Reply:
All I am saying is that you guys should replace `"Enter email address to search"` with `"Enter name or email address to search"`